ecommerce/README.rst

```diff
@@ -0,0 +1,27 @@
+edx Themes for Otto
+===================
+This directory contains themes for Otto, following directory structure should be followed for each theme
+
+
+.. code-block:: text
+
+    my-theme
+        ├── README.rst
+        ├── static
+        |      └── images
+        |      |     └── logo.png
+        |      |
+        |      └── sass
+        |            └── partials
+        |                   └── utilities
+        |                           └── _variables.scss
+        |
+        └── templates
+                └── oscar
+                |     └── dashboard
+                |             └── index.html
+                └── 404.html
+
+A more detailed description of Otto theming and guidelines for theming templates, assets and styles can be found here_.
+
+.. _here: https://github.com/edx/ecommerce/blob/master/docs/theming.rst
```

Comment:
@lamagnifica ecommerce documentation was added after the implementation of theming in ecommerce, so it is accurate. I used this link because documentation there is more personalized for ecommerce. But let me know, if you think it would be better to reference main docs here too, I will update this link.